docs/config-file-ort-yml.md

```diff
@@ -153,7 +153,9 @@ curations:
     concluded_license: "Apache-2.0"
  ```
 
-For findings in third-party dependencies package-configurations can be used to replace findings:
+To overwrite scanner findings by correcting identified licenses in a dependency a package configuration can be used.
```

Comment:
Simpler: "To correct identified licenses in a dependency you can use a package configuration to overwrite scanner findings."